src/databricks/labs/ucx/config.py

```diff
@@ -35,6 +35,7 @@ class WorkspaceConfig:  # pylint: disable=too-many-instance-attributes
 
     override_clusters: dict[str, str] | None = None
     custom_cluster_policy_id: str | None = None
+    num_days_submit_runs_history: int = 30
```

Comment:
Prompt for this value in installer. 

In "new_installer" Integration tests , take 1 instead of 30

Reply:
hey @nfx could you please point me to the correct place - where exactly can I prompt this?

Reply:
upd - found 🆗